.github/workflows/pr_lint_and_test.yml

```diff
@@ -27,11 +27,18 @@ jobs:
         # the dependencies in a separate step
         working-directory: ./examples/cdk
         run: npm ci
+      - name: "Setup SAM"
+        # We use an ad-hoc action so we can specify the SAM CLI version
+        uses: aws-actions/setup-sam@v2
+        with:
+          version: 1.49.0
       - name: Install SAM example packages
         # Since we are not managing the SAM examples with npm workspaces we install
         # the dependencies in a separate step
         working-directory: ./examples/sam
-        run: npm ci
+        run: |
+          sam --version
```

Comment:
(nit) Is this intentional or just for debug?

Reply:
Good catch, I need to remove this